client/lang/src/en.json

```diff
@@ -8,5 +8,7 @@
     "CampaignAdmin.NO_CHANGES": "No changes",
     "CampaignAdmin.PUBLISHCAMPAIGN": "Publish campaign",
     "CampaignAdmin.REMOVED": "Removed",
-    "CampaignAdmin.REVERTCAMPAIGN": "Revert"
-}
\ No newline at end of file
+    "CampaignAdmin.REVERTCAMPAIGN": "Revert",
```

Comment:
Something like: Reverted \"%s\" could give more context as to what is being reverted?

Reply:
As far as I know, that feature hasn't been implemented.